examples/text_classification/example_custom_dataset.py

```diff
@@ -0,0 +1,7 @@
+from autogluon import TextClassification as task
+
+dataset = task.Dataset(filepath='/home/ubuntu/data/tsv/train.csv', usecols=['text', 'target'])
+predictor = task.fit(dataset, epochs=1)
+print('Top-1 val acc: %.3f' % predictor.results['best_reward'])
+test_acc = predictor.evaluate(dataset)
```

Comment:
This example could be better if you also showed how to: 
- read in a separate test dataset and make the predictions/evaluation on it (instead of the same training data)
- reload the trained predictor from disk after task.fit()